### ISSUE TITLE
Mongo Ops Manager/MongoDB is leaving open deleted log files behind

### ISSUE THREAD

**Issue by philrumble-fz** [mongodb/mongodb-enterprise-kubernetes#174]
**What did you do to encounter the bug?**
The PVs are filling up on the primary member due the rotated log files not being released.

**What did you expect?**
I expected the mongo operator to rotate the logs and free up space

**What happened instead?**
Mongo Ops Manager/MongoDB is leaving open deleted log files behind

I’m running Mongo Ops Manager v4.2.6 with a replica set of 3 members.

The primary is constantly filling up on disk usage due to open log files being deleted.

find /proc/*/fd -ls | grep ‘(deleted)’
342635611 0 lr-x------ 1 2000 root 64 Dec 30 03:42 /proc/61/fd/13 -> /var/log/mongodb-mms-automation/mongodb.log.2020-12-29T16-50-02\ (deleted)

I’ve got several hundred of this files. They account for ~ 400Gb of disk space.

the mongo operator is configured to rotate the logs, which it appears to be doing, yet they are staying around.

I’d like a good answer as to why the ops-manager/mongodb is not handling this correctly.

**Screenshots**
![image](https://user-images.githubusercontent.com/67681499/103502960-dcff3480-4e8d-11eb-912b-17a0357484e6.png)

**Operator Information**
 - Operator Version  Mongo Ops Manager v4.2.6 
 - MongoDB Image v4.2.11

**Kubernetes Cluster Information**
 - GKE
 - v.16
 - gcr

If possible, please include:
 k describe mdb mdb-riskselector-rs        
Name:         mdb-riskselector-rs
Namespace:    default
Labels:       app.kubernetes.io/instance=lax-mongodb-databases
Annotations:  <none>
API Version:  mongodb.com/v1
Kind:         MongoDB
Metadata:
  Creation Timestamp:  2020-12-18T03:19:34Z
  Generation:          223
  Resource Version:    221196065
  Self Link:           /apis/mongodb.com/v1/namespaces/default/mongodb/mdb-riskselector-rs
  UID:                 4cb7d7c6-f22d-4a42-b3c4-1acc868e1f3c
Spec:
  Credentials:  om-credentials
  Members:      3
  Ops Manager:
    Config Map Ref:
      Name:    ops-manager-connection
  Persistent:  true
  Pod Spec:
    Node Affinity:
      Required During Scheduling Ignored During Execution:
        Node Selector Terms:
          Match Expressions:
            Key:       cloud.google.com/gke-nodepool
            Operator:  In
            Values:
              database-nodes
    Persistence:
      Single:
        Storage:  500Gi
    Pod Template:
      Metadata:
        Creation Timestamp:  <nil>
      Spec:
        Containers:           <nil>
        Priority Class Name:  priority-high
        Tolerations:
          Effect:    NoSchedule
          Key:       database
          Operator:  Equal
          Value:     true
  Type:              ReplicaSet
  Version:           4.2.11
Status:
  Last Transition:  2021-01-03T20:02:38Z
  Link:             http://ops-manager-svc.default.svc.cluster.local:8080/v2/5fdc1fc95d647a006f9e90d5
  Members:          3
  Phase:            Running
  Type:             ReplicaSet
  Version:          4.2.11
Events:             <none>

**Comment by jamesbroadhead** [comment]
Transferring this to Enterprise, as this relates to Ops Manager

**Comment by antonlisovenko** [comment]
@philrumble-fz can you please show the output from `ls -la /var/log/mongodb-mms-automation/mongodb.log.*`?
Also could you tell the Operator version?
Please also provide the following
`kubectl get mdb <name> -o yaml`
`kubectl get pod <name-0> -o yaml`

I took some long running instance (with the same log settings as shown by you) and I see that  all old files are archived (which is expected)
```
I have no name!@my-replica-set-0:/$ ls -la /var/log/mongodb-mms-automation/mongodb.log.*
-rw-r--r-- 1 2000 root       23 Nov 19 10:22 /var/log/mongodb-mms-automation/mongodb.log.2020-11-19T10-23-09.gz
-rw-r--r-- 1 2000 root    34043 Nov 20 10:23 /var/log/mongodb-mms-automation/mongodb.log.2020-11-20T10-23-09.gz
-rw-r--r-- 1 2000 root    24391 Nov 21 10:24 /var/log/mongodb-mms-automation/mongodb.log.2020-11-21T10-24-13.gz
-rw-r--r-- 1 2000 root    24248 Nov 22 10:25 /var/log/mongodb-mms-automation/mongodb.log.2020-11-22T10-25-53.gz
-rw-r--r-- 1 2000 root    24056 Nov 23 10:26 /var/log/mongodb-mms-automation/mongodb.log.2020-11-23T10-26-56.gz
-rw-r--r-- 1 2000 root    24825 Nov 24 10:28 /var/log/mongodb-mms-automation/mongodb.log.2020-11-24T10-28-01.gz
-rw-r--r-- 1 2000 root    24778 Nov 25 10:29 /var/log/mongodb-mms-automation/mongodb.log.2020-11-25T10-29-52.gz
-rw-r--r-- 1 2000 root    24731 Nov 26 10:30 /var/log/mongodb-mms-automation/mongodb.log.2020-11-26T10-30-50.gz
-rw-r--r-- 1 2000 root    24241 Nov 27 10:31 /var/log/mongodb-mms-automation/mongodb.log.2020-11-27T10-31-58.gz
-rw-r--r-- 1 2000 root    24879 Nov 28 10:33 /var/log/mongodb-mms-automation/mongodb.log.2020-11-28T10-33-46.gz
-rw-r--r-- 1 2000 root    24181 Nov 29 10:34 /var/log/mongodb-mms-automation/mongodb.log.2020-11-29T10-34-54.gz
-rw-r--r-- 1 2000 root    24517 Nov 30 10:35 /var/log/mongodb-mms-automation/mongodb.log.2020-11-30T10-35-58.gz
-rw-r--r-- 1 2000 root    25690 Dec  1 10:37 /var/log/mongodb-mms-automation/mongodb.log.2020-12-01T10-37-38.gz
-rw-r--r-- 1 2000 root    24459 Dec  2 10:38 /var/log/mongodb-mms-automation/mongodb.log.2020-12-02T10-38-45.gz
-rw-r--r-- 1 2000 root    25061 Dec  3 10:39 /var/log/mongodb-mms-automation/mongodb.log.2020-12-03T10-39-57.gz
-rw-r--r-- 1 2000 root    27176 Dec  4 10:41 /var/log/mongodb-mms-automation/mongodb.log.2020-12-04T10-41-43.gz
-rw-r--r-- 1 2000 root    24063 Dec  5 10:42 /var/log/mongodb-mms-automation/mongodb.log.2020-12-05T10-42-38.gz
-rw-r--r-- 1 2000 root    24892 Dec  6 10:44 /var/log/mongodb-mms-automation/mongodb.log.2020-12-06T10-44-18.gz
-rw-r--r-- 1 2000 root    24309 Dec  7 10:45 /var/log/mongodb-mms-automation/mongodb.log.2020-12-07T10-45-26.gz
-rw-r--r-- 1 2000 root    24643 Dec  8 10:46 /var/log/mongodb-mms-automation/mongodb.log.2020-12-08T10-46-35.gz
-rw-r--r-- 1 2000 root    24915 Dec  9 10:48 /var/log/mongodb-mms-automation/mongodb.log.2020-12-09T10-48-18.gz
-rw-r--r-- 1 2000 root    24680 Dec 10 10:49 /var/log/mongodb-mms-automation/mongodb.log.2020-12-10T10-49-17.gz
-rw-r--r-- 1 2000 root    24180 Dec 11 10:50 /var/log/mongodb-mms-automation/mongodb.log.2020-12-11T10-50-32.gz
-rw-r--r-- 1 2000 root    26319 Dec 12 10:52 /var/log/mongodb-mms-automation/mongodb.log.2020-12-12T10-52-13.gz
-rw-r--r-- 1 2000 root    24450 Dec 13 10:53 /var/log/mongodb-mms-automation/mongodb.log.2020-12-13T10-53-22.gz
-rw-r--r-- 1 2000 root    24606 Dec 14 10:54 /var/log/mongodb-mms-automation/mongodb.log.2020-12-14T10-54-33.gz
-rw-r--r-- 1 2000 root    26076 Dec 15 10:56 /var/log/mongodb-mms-automation/mongodb.log.2020-12-15T10-56-02.gz
-rw-r--r-- 1 2000 root   641151 Dec 16 10:56 /var/log/mongodb-mms-automation/mongodb.log.2020-12-16T10-56-41.gz
-rw-r--r-- 1 2000 root  1084069 Dec 17 10:56 /var/log/mongodb-mms-automation/mongodb.log.2020-12-17T10-56-46.gz
-rw-r--r-- 1 2000 root  1007046 Dec 18 10:56 /var/log/mongodb-mms-automation/mongodb.log.2020-12-18T10-56-52.gz
-rw-r--r-- 1 2000 root  1016456 Dec 19 10:56 /var/log/mongodb-mms-automation/mongodb.log.2020-12-19T10-56-58.gz
-rw-r--r-- 1 2000 root  1005834 Dec 20 10:59 /var/log/mongodb-mms-automation/mongodb.log.2020-12-20T10-59-06.gz
-rw-r--r-- 1 2000 root  1011586 Dec 21 10:59 /var/log/mongodb-mms-automation/mongodb.log.2020-12-21T10-59-13.gz
-rw-r--r-- 1 2000 root  1018291 Dec 22 10:59 /var/log/mongodb-mms-automation/mongodb.log.2020-12-22T10-59-20.gz
-rw-r--r-- 1 2000 root  1023328 Dec 23 10:59 /var/log/mongodb-mms-automation/mongodb.log.2020-12-23T10-59-27.gz
-rw-r--r-- 1 2000 root  1022603 Dec 24 10:59 /var/log/mongodb-mms-automation/mongodb.log.2020-12-24T10-59-35.gz
-rw-r--r-- 1 2000 root  1030955 Dec 25 10:59 /var/log/mongodb-mms-automation/mongodb.log.2020-12-25T10-59-42.gz
-rw-r--r-- 1 2000 root  1025581 Dec 26 10:59 /var/log/mongodb-mms-automation/mongodb.log.2020-12-26T10-59-49.gz
-rw-r--r-- 1 2000 root  1032306 Dec 27 10:59 /var/log/mongodb-mms-automation/mongodb.log.2020-12-27T10-59-56.gz
-rw-r--r-- 1 2000 root  1019170 Dec 28 11:00 /var/log/mongodb-mms-automation/mongodb.log.2020-12-28T11-00-03.gz
-rw-r--r-- 1 2000 root  1034225 Dec 29 11:00 /var/log/mongodb-mms-automation/mongodb.log.2020-12-29T11-00-10.gz
-rw-r--r-- 1 2000 root  1029937 Dec 30 11:00 /var/log/mongodb-mms-automation/mongodb.log.2020-12-30T11-00-18.gz
-rw-r--r-- 1 2000 root  1027553 Dec 31 11:00 /var/log/mongodb-mms-automation/mongodb.log.2020-12-31T11-00-26.gz
-rw------- 1 2000 root 39877126 Jan  1 11:01 /var/log/mongodb-mms-automation/mongodb.log.2021-01-01T11-01-32
-rw------- 1 2000 root 39822314 Jan  2 11:01 /var/log/mongodb-mms-automation/mongodb.log.2021-01-02T11-01-40
-rw------- 1 2000 root 39810464 Jan  3 11:01 /var/log/mongodb-mms-automation/mongodb.log.2021-01-03T11-01-47
-rw------- 1 2000 root 39814690 Jan  4 11:01 /var/log/mongodb-mms-automation/mongodb.log.2021-01-04T11-01-55
```

**Comment by philrumble-fz** [comment]
We do have datadog-agents running on the nodes, though they should not interfere with any files.
The operator version is v4.2.6

as requested; 
ls -la /var/log/mongodb-mms-automation/mongodb.log.*
-rw-r--r-- 1 2000 2000  106838181 Jan  4 22:34 /var/log/mongodb-mms-automation/mongodb.log.2021-01-04T22-34-43.gz
-rw-r--r-- 1 2000 2000  106854015 Jan  4 22:38 /var/log/mongodb-mms-automation/mongodb.log.2021-01-04T22-38-32.gz
-rw-r--r-- 1 2000 2000  110307998 Jan  4 22:42 /var/log/mongodb-mms-automation/mongodb.log.2021-01-04T22-42-19.gz
-rw-r--r-- 1 2000 2000  109847035 Jan  4 22:46 /var/log/mongodb-mms-automation/mongodb.log.2021-01-04T22-46-07.gz
-rw-r--r-- 1 2000 2000  108585602 Jan  4 22:49 /var/log/mongodb-mms-automation/mongodb.log.2021-01-04T22-49-54.gz
-rw-r--r-- 1 2000 2000  108034847 Jan  4 22:53 /var/log/mongodb-mms-automation/mongodb.log.2021-01-04T22-53-41.gz
-rw-r--r-- 1 2000 2000  109191289 Jan  4 22:57 /var/log/mongodb-mms-automation/mongodb.log.2021-01-04T22-57-28.gz
-rw-r--r-- 1 2000 2000  109209139 Jan  4 23:01 /var/log/mongodb-mms-automation/mongodb.log.2021-01-04T23-01-17.gz
-rw-r--r-- 1 2000 2000  109484161 Jan  4 23:05 /var/log/mongodb-mms-automation/mongodb.log.2021-01-04T23-05-05.gz
-rw-r--r-- 1 2000 2000  108475612 Jan  4 23:08 /var/log/mongodb-mms-automation/mongodb.log.2021-01-04T23-08-53.gz
-rw-r--r-- 1 2000 2000  109580578 Jan  4 23:12 /var/log/mongodb-mms-automation/mongodb.log.2021-01-04T23-12-41.gz
-rw-r--r-- 1 2000 2000  109871184 Jan  4 23:16 /var/log/mongodb-mms-automation/mongodb.log.2021-01-04T23-16-28.gz
-rw-r--r-- 1 2000 2000  108851974 Jan  4 23:20 /var/log/mongodb-mms-automation/mongodb.log.2021-01-04T23-20-18.gz
-rw-r--r-- 1 2000 2000  108469840 Jan  4 23:24 /var/log/mongodb-mms-automation/mongodb.log.2021-01-04T23-24-06.gz
-rw-r--r-- 1 2000 2000  107707845 Jan  4 23:27 /var/log/mongodb-mms-automation/mongodb.log.2021-01-04T23-27-54.gz
-rw-r--r-- 1 2000 2000  108677701 Jan  4 23:31 /var/log/mongodb-mms-automation/mongodb.log.2021-01-04T23-31-42.gz
-rw------- 1 2000 2000 2154689655 Jan  4 23:35 /var/log/mongodb-mms-automation/mongodb.log.2021-01-04T23-35-31
-rw------- 1 2000 2000 2147990195 Jan  4 23:39 /var/log/mongodb-mms-automation/mongodb.log.2021-01-04T23-39-19
-rw------- 1 2000 2000  900935680 Jan  4 23:40 /var/log/mongodb-mms-automation/mongodb.log.2021-01-04T23-41-08
-rw------- 1 2000 2000          0 Jan  4 23:41 /var/log/mongodb-mms-automation/mongodb.log.2021-01-04T23-42-19


k get mdb mdb-riskselector-rs -o yaml
apiVersion: mongodb.com/v1
kind: MongoDB
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"mongodb.com/v1","kind":"MongoDB","metadata":{"annotations":{},"labels":{"app.kubernetes.io/instance":"lax-mongodb-databases"},"name":"mdb-riskselector-rs","namespace":"default"},"spec":{"credentials":"om-credentials","members":3,"opsManager":{"configMapRef":{"name":"ops-manager-connection"}},"persistent":true,"podSpec":{"nodeAffinity":{"requiredDuringSchedulingIgnoredDuringExecution":{"nodeSelectorTerms":[{"matchExpressions":[{"key":"cloud.google.com/gke-nodepool","operator":"In","values":["database-nodes"]}]}]}},"persistence":{"single":{"storage":"500Gi"}},"podTemplate":{"spec":{"priorityClassName":"priority-high","tolerations":[{"effect":"NoSchedule","key":"database","operator":"Equal","value":"true"}]}}},"type":"ReplicaSet","version":"4.2.11"}}
  creationTimestamp: "2020-12-18T03:19:34Z"
  generation: 231
  labels:
    app.kubernetes.io/instance: lax-mongodb-databases
  name: mdb-riskselector-rs
  namespace: default
  resourceVersion: "222156690"
  selfLink: /apis/mongodb.com/v1/namespaces/default/mongodb/mdb-riskselector-rs
  uid: 4cb7d7c6-f22d-4a42-b3c4-1acc868e1f3c
spec:
  credentials: om-credentials
  members: 3
  opsManager:
    configMapRef:
      name: ops-manager-connection
  persistent: true
  podSpec:
    nodeAffinity:
      requiredDuringSchedulingIgnoredDuringExecution:
        nodeSelectorTerms:
        - matchExpressions:
          - key: cloud.google.com/gke-nodepool
            operator: In
            values:
            - database-nodes
    persistence:
      single:
        storage: 500Gi
    podTemplate:
      metadata:
        creationTimestamp: null
      spec:
        containers: null
        priorityClassName: priority-high
        tolerations:
        - effect: NoSchedule
          key: database
          operator: Equal
          value: "true"
  type: ReplicaSet
  version: 4.2.11
status:
  lastTransition: "2021-01-04T09:50:50Z"
  link: http://ops-manager-svc.default.svc.cluster.local:8080/v2/5fdc1fc95d647a006f9e90d5
  members: 3
  phase: Running
  type: ReplicaSet
  version: 4.2.11



k get pod mdb-riskselector-rs-0 -o yaml       
apiVersion: v1
kind: Pod
metadata:
  annotations:
    certHash: ""
    kubernetes.io/limit-ranger: 'LimitRanger plugin set: cpu request for container
      mongodb-enterprise-database'
  creationTimestamp: "2020-12-18T03:33:58Z"
  generateName: mdb-riskselector-rs-
  labels:
    app: mdb-riskselector-rs-svc
    controller: mongodb-enterprise-operator
    controller-revision-hash: mdb-riskselector-rs-7fc8f5bd9d
    pod-anti-affinity: mdb-riskselector-rs
    statefulset.kubernetes.io/pod-name: mdb-riskselector-rs-0
  name: mdb-riskselector-rs-0
  namespace: default
  ownerReferences:
  - apiVersion: apps/v1
    blockOwnerDeletion: true
    controller: true
    kind: StatefulSet
    name: mdb-riskselector-rs
    uid: 7ef5a92c-ee2a-4f37-a255-e1dcca40c546
  resourceVersion: "206222349"
  selfLink: /api/v1/namespaces/default/pods/mdb-riskselector-rs-0
  uid: 97a34637-80f4-4c20-b669-6a4d37ee95b1
spec:
  affinity:
    nodeAffinity:
      requiredDuringSchedulingIgnoredDuringExecution:
        nodeSelectorTerms:
        - matchExpressions:
          - key: cloud.google.com/gke-nodepool
            operator: In
            values:
            - database-nodes
    podAntiAffinity:
      preferredDuringSchedulingIgnoredDuringExecution:
      - podAffinityTerm:
          labelSelector:
            matchLabels:
              pod-anti-affinity: mdb-riskselector-rs
          topologyKey: kubernetes.io/hostname
        weight: 100
  containers:
  - env:
    - name: AGENT_API_KEY
      value: ********
    - name: BASE_URL
      value: http://ops-manager-svc.default.svc.cluster.local:8080
    - name: GROUP_ID
      value: ****
    - name: LOG_LEVEL
    - name: SSL_REQUIRE_VALID_MMS_CERTIFICATES
      value: "true"
    - name: USER_LOGIN
      value: ****
    image: quay.io/mongodb/mongodb-enterprise-database:1.4.4
    imagePullPolicy: Always
    livenessProbe:
      exec:
        command:
        - /mongodb-automation/files/probe.sh
      failureThreshold: 6
      initialDelaySeconds: 60
      periodSeconds: 30
      successThreshold: 1
      timeoutSeconds: 30
    name: mongodb-enterprise-database
    ports:
    - containerPort: 27017
      protocol: TCP
    readinessProbe:
      exec:
        command:
        - /mongodb-automation/files/readinessprobe
      failureThreshold: 240
      initialDelaySeconds: 5
      periodSeconds: 5
      successThreshold: 1
      timeoutSeconds: 1
    resources:
      requests:
        cpu: 100m
    securityContext:
      runAsNonRoot: true
      runAsUser: 2000
    terminationMessagePath: /dev/termination-log
    terminationMessagePolicy: File
    volumeMounts:
    - mountPath: /data
      name: data
      subPath: data
    - mountPath: /journal
      name: data
      subPath: journal
    - mountPath: /var/log/mongodb-mms-automation
      name: data
      subPath: logs
    - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
      name: mongodb-enterprise-database-pods-token-4m4tb
      readOnly: true
  dnsPolicy: ClusterFirst
  enableServiceLinks: true
  hostname: mdb-riskselector-rs-0
  nodeName: gke-cluster-lax-database-nodes-0012210f-bvcd
  priority: 1000
  priorityClassName: priority-high
  restartPolicy: Always
  schedulerName: default-scheduler
  securityContext:
    fsGroup: 2000
  serviceAccount: mongodb-enterprise-database-pods
  serviceAccountName: mongodb-enterprise-database-pods
  subdomain: mdb-riskselector-rs-svc
  terminationGracePeriodSeconds: 600
  tolerations:
  - effect: NoSchedule
    key: database
    operator: Equal
    value: "true"
  - effect: NoExecute
    key: node.kubernetes.io/not-ready
    operator: Exists
    tolerationSeconds: 300
  - effect: NoExecute
    key: node.kubernetes.io/unreachable
    operator: Exists
    tolerationSeconds: 300
  volumes:
  - name: data
    persistentVolumeClaim:
      claimName: data-mdb-riskselector-rs-0
  - name: mongodb-enterprise-database-pods-token-4m4tb
    secret:
      defaultMode: 420
      secretName: mongodb-enterprise-database-pods-token-4m4tb
status:
  conditions:
  - lastProbeTime: null
    lastTransitionTime: "2020-12-18T03:33:58Z"
    status: "True"
    type: Initialized
  - lastProbeTime: null
    lastTransitionTime: "2020-12-25T16:37:29Z"
    status: "True"
    type: Ready
  - lastProbeTime: null
    lastTransitionTime: "2020-12-25T16:37:29Z"
    status: "True"
    type: ContainersReady
  - lastProbeTime: null
    lastTransitionTime: "2020-12-18T03:33:58Z"
    status: "True"
    type: PodScheduled
  containerStatuses:
  - containerID: docker://837c3876312b074dd2b7b69a0f95834119465473f36aba5260b8da7f20306e48
    image: quay.io/mongodb/mongodb-enterprise-database:1.4.4
    imageID: docker-pullable://quay.io/mongodb/mongodb-enterprise-database@sha256:844e124c787dbc58b7189e3858f6c0da7a4fe268f9da0dea688c95fcb23a037d
    lastState:
      terminated:
        containerID: docker://38a6c09fa49d2f8377067990b102bccba6e36dcaa376d17de4c03f3310b2e934
        exitCode: 1
        finishedAt: "2020-12-25T16:36:50Z"
        reason: OOMKilled
        startedAt: "2020-12-18T03:34:18Z"
    name: mongodb-enterprise-database
    ready: true
    restartCount: 1
    started: true
    state:
      running:
        startedAt: "2020-12-25T16:36:51Z"
  hostIP: 10.168.0.113
  phase: Running
  podIP: 10.56.68.5
  podIPs:
  - ip: 10.56.68.5
  qosClass: Burstable
  startTime: "2020-12-18T03:33:58Z"

**Comment by antonlisovenko** [comment]
Thanks @philrumble-fz 

I failed to reproduce this after tweaking the logging parameters and playing with files sizes - the extra log files were removed without any problems and no pending file descriptors were left.

Looking into the files you've provided I see that the logging speed seems to be quite high - the tar gz files are created each 2-4 minutes, also I see that the removal operation failed on the normal file, not the compressed one, so this could be some race condition happening under the load.

In conclusion - I'm not sure if the Operator team can help you more on this and I'd recommend you to file the support ticket using https://support.mongodb.com/ and you'll definitely get more support! Please reference this issue from there (and would be great to reformat the output from the `kubectl` using code markup ("```")

**Comment by jamesbroadhead** [comment]
In order to help us manage issues and requests we have switched from GitHub Issues to the [MongoDB Support Center](https://support.mongodb.com/). Please use your Enterprise support account to raise your requests. 

If this issue is still affecting you, please open a [Support Case](https://support.mongodb.com/) or for non-blocking issues, you can make a [Feature Request](https://feedback.mongodb.com/forums/924355-ops-tools)
